### DISCUSSION TITLE
dim OS window when it loses focus

### DISCUSSION THREAD

**Discussion by msackman** [kovidgoyal/kitty#7124] [Ideas]
One feature I'm really missing from konsole is the ability to dim the entire window when it loses focus. For those of us using tiling window managers, possibly with titlebars and borders disabled, the only indication in kitty as to which window has the focus is the shape of the caret.
Having searched through the kitty docs, I can't see any way to react to focus or unfocus events. Am I missing something, or is this not something that's been considered so far - I didn't find any other mention of this in the github issues or discussion?

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.inactive_text_alpha

**Comment by msackman** [comment]
That'll do nicely - thank you. Sorry - I was searching the docs for "focus" which is why I missed it.
